rasa/cli/initial_project/tests/conversation_tests.md

```diff
@@ -0,0 +1,40 @@
+## happy path 1
+* greet: hello there!
+  - utter_greet
+* mood_great: amazing
+  - utter_happy
+  
+## sad path 1
+* greet: hello
+  - utter_greet
+* mood_unhappy: not good
+  - utter_cheer_up
+  - utter_did_that_help
+* affirm: yes
+  - utter_happy
+  
+## sad path 2
+* greet: hello
+  - utter_greet
+* mood_unhappy: not good
+  - utter_cheer_up
+  - utter_did_that_help
+* deny: not really
+  - utter_goodbye
+  
+## sad path 3
```

Comment:
should we also add a story which differs from the stories in the training data? E.g. one including `goodbye` at the end? Could be confusing for new users, that these stories are just the training stories with a few more annotations

Reply:
Done!